README.md

```diff
@@ -72,6 +72,7 @@ The object literal follows the format:
 	cookie: 				// session cookie configuration
 	getUserString:			// method to return a string id for a user
 
+	noAccessLogs: false,			// disables access logs
```

Comment:
I would much prefer that this be an enable flag rather than a disable flag, so that true -> on and false -> off. What do you think about that?